build/Dockerfile

```diff
@@ -67,10 +67,17 @@ exec ${HOME}/testing/run "$@" \n\
 # Prepare the image for must-gather support
 RUN mkdir /must-gather && chmod 777 /must-gather
 
+# Prepare osde2e results folder
+RUN mkdir /test-run-results && chmod 777 /test-run-results
```

Comment:
nit: Can just do `mkdir --mode 777 /test-run-results`

Reply:
nit: kept the same format used in the file. But yes - we can do that :)

Reply:
ah my eyes only see the green stuff :upside_down_face: